sdk/communication/communication-common/review/communication-common.api.md

```diff
@@ -108,6 +111,21 @@ export interface PhoneNumberKind extends PhoneNumberIdentifier {
     kind: "PhoneNumber";
 }
 
+// @internal
+export const _serializeCommunicationIdentifier: (identifier: CommunicationIdentifier) => _SerializedCommunicationIdentifier;
+
+// @internal
+export interface _SerializedCommunicationIdentifier {
+    id?: string;
+    isAnonymous?: boolean;
+    kind: _SerializedCommunicationIdentifierKind;
+    microsoftTeamsUserId?: string;
+    phoneNumber?: string;
+}
+
+// @internal
+export type _SerializedCommunicationIdentifierKind = "unknown" | "communicationUser" | "phoneNumber" | "callingApplication" | "microsoftTeamsUser";
```

Comment:
Unfortunately, the same kind values in the SDK models are PascalCase while the swagger uses lowerCamelCase. We should probably change the SDK model to use lowerCamelCase and break the API while we're still in preview.

Thoughts?

Reply:
JS aligning to REST does make sense to me, but I am no JS expert